### PR TITLE
Add .env.production-goerli

### PR DESCRIPTION
**Pull request by snario**
Beginning to add Goerli support.

TODO:

- [ ] Send ETH to the hub address
- [ ] Send ETH to the e2e test address
- [ ] Change hard-coded "ropsten" strings in the codebase

### CODE REVIEW

**Comment by snario** [comment]
Deployment logs: https://goerli.etherscan.io/address/0xea23b0654b1871c1932fbdd6e059843b0d42d9fb

File: .env.production-goerli [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+LOG_LEVEL = 'info'
+# e2e-tests
+BROWSER_LOG_DESTINATION = 'browser.log'
+HEADLESS = 'true'
+USE_DAPPETEER = 'false'
+WEB3TORRENT_URL = 'https://web3torrent.statechannels.org'
+CLOSE_BROWSERS = 'true'
+
+## web3torrent, xstate-wallet, rps, simple-hub
+CHAIN_NETWORK_ID = '3'
```
Comment (geoknee) [inline .env.production-goerli:13]: `5` I believe: https://github.com/goerli/testnet

Comment (snario) [inline .env.production-goerli:13]:
```suggestion
CHAIN_NETWORK_ID = '5'
```

**Comment by geoknee** [comment]
For future reference, this was the best faucet I found (no social media stuff necessary) https://goerli-faucet.slock.it/

**Comment by lalexgap** [comment]
Closing in favour of #2151

**Pull request closed without merging**